### PR TITLE
Update CI

### PR DESCRIPTION
**Pull request by gwynne**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-  pull_request: { types: [opened, reopened, synchronize, ready_for_review] }
-  push: { branches: [ main ] }
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
+on:
+  pull_request: { types: [opened, reopened, synchronize, ready_for_review] }
+  push: { branches: [ main ] }
 env:
-  LOG_LEVEL: into
```
Comment (MahdiBM) [inline .github/workflows/test.yml:9]: LUL

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -41,40 +41,39 @@ jobs:
...
-          - mariadb:10.4
-          - mariadb:10.11
+          - mariadb:10.2
+          - mariadb:11
+          - percona:5.7
           - percona:8.0
         swiftver:
-          - swift:5.6-focal
           - swift:5.7-jammy
           - swift:5.8-jammy
```
Comment (MahdiBM) [inline .github/workflows/test.yml:50]: Why not have have bionic/focal too? Doesn't hurt I guess.

**Review by MahdiBM — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (gwynne) [reply .github/workflows/test.yml:9]: Yeah already fixed that 🤣

File: .github/workflows/test.yml (hunk 2 above)
Comment (gwynne) [reply .github/workflows/test.yml:50]: 'cause truth is specific Ubuntu flavor matters to basically nothing any of Vapor's packages do and the rewrite branch is about to drop 5.6 support anyway (don't even talk to me about how tempted I am to drop 5.7 too >.<)

**Pull request merged**